pkg/kubelet/pleg/generic.go

```diff
@@ -402,10 +397,6 @@ func computeEvents(logger klog.Logger, oldPod, newPod *kubecontainer.Pod, cid *k
 	return generateEvents(logger, pid, cid.ID, oldState, newState)
 }
 
-func (g *GenericPLEG) cacheEnabled() bool {
```

Comment:
we at least need to check for nil in constructor.

Reply:
Done. I just put a panic in to fail fast. Or do you think I need to propogate an error instead?